receiver/mongodbatlasreceiver/receiver_test.go

```diff
@@ -71,3 +71,64 @@ func TestTimeConstraints(t *testing.T) {
 		t.Run(testCase.name, testCase.run)
 	}
 }
+
+func TestShouldProcessCluster(t *testing.T) {
+	tests := []struct {
+		name        string
+		projectCfg  *ProjectConfig
+		clusterName string
+		want        bool
+	}{
+		{
+			name: "included cluster",
```

Comment:
Add a test case for include list, but cluster not in list, and exclude list, but cluster not in list.

Reply:
The functions mentioned above should make it more clear that this combination is not possible.